.gitignore

```diff
@@ -2,3 +2,5 @@ _output/
 /shp
 /shp-*
 /bin/*
+.idea/
```

Comment:
This should go in your global git ignore file, not here.
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
fixed; thanks for letting me learn about this feature!